app/components/api-utils.js

```diff
@@ -4,6 +4,7 @@ const {endsWith} = require('lodash');
 
 const fetch = require('node-fetch');
 const HttpsProxyAgent = require('https-proxy-agent');
+const config = require('app/config.js');
```

Comment:
The require statement doesn't need the .js at the end of the file path so this could be changed to 

`const config = require('app/config');`